public/js/pimcore/object/classes/data/password.js

```diff
@@ -110,7 +110,7 @@ pimcore.object.classes.data.password = Class.create(pimcore.object.classes.data.
             const saltCombo = new Ext.form.field.ComboBox({
                 xtype: "combo",
                 width: 300,
-                fieldLabel: t("saltlocation"),
+                fieldLabel: t("saltlocation") + t("(deprecated)"),
```

Comment:
Why not using the existing deprecated translation key?
```suggestion
                fieldLabel: t('saltlocation') + '(' + t('deprecated') + ')',
```

Reply:
Wanted to use something new that could be hidden entirely, if you change the translation key value to " ", if i use the suggested one with concatenated text, it would be "( )".

Maybe it shouldn't be a matter because it should be a field visible to superuser, but in case it gets annoying for UI purposes.